packages/cli/src/lib/flags.ts

```diff
@@ -20,6 +21,19 @@ export const commonFlags = {
     env: 'SHOPIFY_HYDROGEN_FLAG_FORCE',
     char: 'f',
   }),
+  shop: Flags.string({
+    char: 's',
+    description:
+      'Shop URL. It can be the shop prefix (janes-apparel)' +
+      ' or the full myshopify.com URL (janes-apparel.myshopify.com, https://janes-apparel.myshopify.com).',
+    env: 'SHOPIFY_SHOP',
+    parse: async (input) => normalizeStoreFqdn(input),
+  }),
+  hydrogenStorefront: Flags.string({
+    char: 'h',
+    description: 'The name of a Hydrogen Storefront (e.g. "Jane\'s Apparel")',
+    env: 'SHOPIFY_HYDROGEN_STOREFRONT',
+  }),
```

Comment:
Do we need these as "common flags" for every command?

Reply:
Good question! I thought I had a reason for this but it might only be necessary for the `link` command so I might move it back there.

Reply:
Moved it into the `link` command. If other commands end up using it we can think about moving it over then.